test/test_rules_against_dialect_fixtures.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+echo "This script will run 'sqlfluff fix -f' on all our dialect fixtures"
+echo "to look for critical errors."
+echo "WARNING this will change the fixtures so do not commit these changes!"
+
+DIALECTS=($( ls -d test/fixtures/dialects/*/ ))
+for DIALECT in "${DIALECTS[@]}"
+do
+    echo "Testing $DIALECT SQL files fix without critical errors..."
+    OUTPUT=$(sqlfluff fix -f ${DIALECT})
```

Comment:
Interested in adding other `.sqlfluff` files, e.g. Danny's from #2811? (Feel free to create a ticket for later.)

Reply:
It's already the longest running job so would suggest a separate, parallel, job if we do that.

However I did test it with `tab_space = 2` and got no CRITICAL errors (which is all it currently looks for) any more since your fix, plus the ones here.

Are you still getting the errors if you do it on this branch? Cause if not, then no need to add. The default config identified the issues and they were fixed.